### ISSUE TITLE
[Enhancement] Allow supervisors to re-open closed work orders

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#5397]
As a supervisor, I need the ability to re-open a closed work order so that I can make corrections to the work order.

However, "lead" technicians who are responsible for closing work orders should not have the ability to re-open work orders.

This need has come up specifically to address inventory-related corrections, where errors are fairly common.

**Comment by johnclary** [comment]
Done.

I created a new `Tech Supervisor | AMD` role and renamed the old `Supervisor | AMD` role to `Lead Tech | AMD`.

Users with the `Tech Supervisor | AMD` are presented with a "Re-Open Work Order" button if a work order is closed.

I gave this new role to Billy, Frank, and Nick.


<img width="1169" alt="Screen Shot 2021-03-04 at 11 03 26 PM" src="https://user-images.githubusercontent.com/14793120/110070053-bbf87a00-7d3e-11eb-97a3-2ec0ae68f4e4.png">
